World_Simulation/auto_gauges.py

```diff
@@ -0,0 +1,243 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+import os
+
+import numpy as np
+import math
+
+from clawpack.geoclaw.surge.storm import Storm
+import clawpack.clawutil as clawutil
+from clawpack.geoclaw import topotools
+
+##### 
+# to automatically place your gauges, first add your topography
+# and storm files below. Then, in setrun, import this module ("import auto_gauges")
+# and call "auto_gauges.return_gauge_points(rundata)" where gauges are supposed to go
+#####
+
+## directory where your data is
+DATA = os.path.join(os.environ.get('DATA_DIR', os.getcwd()))
+
+####
+# make topography file(s) into topo objects (topotools.Topography()) and
+# add to topofiles list
+####
+topofiles = []
+
+for n in [2,3,12,13,14]:
+    topo = topotools.Topography()
+    topo.read('../topo_files/world_' + str(n) + '.tt3', topo_type=3) # this is for world topo, put the path to your files
+    topofiles.append(topo)
+
+## replace with your storm
+atcf_path = os.path.join(DATA, "bal162007.dat")
```

Comment:
Maybe instead of having the storm as a global variable this should be an argument to each of the functions.